vendor/assets/stylesheets/dvl/components/page_header.scss

```diff
@@ -33,35 +36,92 @@
   }
 
   > .button,
-  > form .button {
+  > form .button,
```

Comment:
Ah yeah, you're right. Do you think we need the direct child selector (`>`), or can we remove it to simplify things? (I guess I could test by adding an example to `forms.rb` or a new regression pgae.)